sentry/src/main/java/io/sentry/Baggage.java

```diff
@@ -44,13 +45,13 @@ protected DecimalFormat initialValue() {
   private static final DecimalFormatterThreadLocal decimalFormatter =
       new DecimalFormatterThreadLocal();
 
-  final @NotNull Map<String, String> keyValues;
-  @Nullable Double sampleRate;
-  @Nullable Double sampleRand;
+  private final @NotNull Map<String, String> keyValues;
```

Comment:
```suggestion
  private final @NotNull ConcurrentHashMap<String, String> keyValues;
```

we should also change this in all locations of this file, where we have `Map` as a param or create a new `HashMap`. e.g. ctors and fromHeader